<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_choixdate</class>
 <widget class="QDialog" name="dlg_choixdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>171</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>357</width>
    <height>171</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>357</width>
    <height>171</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDateEdit" name="DepuisdateEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="JusquAdateEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="DategroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Choix de la période</string>
   </property>
   <widget class="QRadioButton" name="JourradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SemaineradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Semaine</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MoisradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mois</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AnneeradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Année</string>
    </property>
   </widget>
  </widget>
  <widget class="UpPushButton" name="OKupPushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>115</y>
     <width>105</width>
     <height>46</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset resource="../assets/Images/images.qrc">
     <normaloff>:/start.png</normaloff>:/start.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Depuislabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Depuis le :</string>
   </property>
  </widget>
  <widget class="QLabel" name="JusquAulabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Jusqu'au:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PlusDateDebutpushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>25</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../assets/Images/images.qrc">
     <normaloff>:/up.png</normaloff>:/up.png</iconset>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="MoinsDateDebutpushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../assets/Images/images.qrc">
     <normaloff>:/down.png</normaloff>:/down.png</iconset>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PlusDateFinpushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>75</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../assets/Images/images.qrc">
     <normaloff>:/up.png</normaloff>:/up.png</iconset>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="MoinsDateFinpushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../assets/Images/images.qrc">
     <normaloff>:/down.png</normaloff>:/down.png</iconset>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="MoinsFinPeriodepushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>30</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../assets/Images/images.qrc">
     <normaloff>:/down.png</normaloff>:/down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PlusDebutPeriodepushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>30</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../assets/Images/images.qrc">
     <normaloff>:/up.png</normaloff>:/up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="UpPushButton" name="AnnulupPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>115</y>
     <width>105</width>
     <height>46</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="icon">
    <iconset resource="../assets/Images/images.qrc">
     <normaloff>:/Supprime.png</normaloff>:/Supprime.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpPushButton</class>
   <extends>QPushButton</extends>
   <header>uppushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../assets/Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
